index.d.ts

```diff
@@ -658,6 +664,11 @@ declare module '@primer/components/src/Fixed' {
   export default Fixed
 }
 
+declare module '@primer/components/src/Position' {
+  import {Relative, Absolute, Sticky, Fixed} from '@primer/components'
+  export {Relative, Absolute, Sticky, Fixed}
+}
```

Comment:
@dmarcey do these modules look right? previously we were exporting each one from `@primer/components/src/Relative`, etc but that's incorrect because there's not actually a file there (they are all exported from on Position file) similar to how we do it for Heading

Reply:
NB: It’s possible to combine the `import` and `export` into a single line:
```
export { Relative, Absolute, Sticky, Fixed } from "@primer/components";
```

Reply:
That looks like it should work! @smockle 's suggestion seems like it simplifies things a little bit too.